components/admin/requests/physician-assessment/Form.tsx

```diff
@@ -121,6 +133,79 @@ export default function PhysicianAssessmentForm({
 
       <Divider borderColor="border.secondary" />
 
+      {/* Mobility Aids Section */}
+
+      <Box paddingY="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {'Mobility Aids'}
+        </Text>
+
+        <Stack spacing="20px">
+          <FormControl isRequired paddingBottom="24px">
+            <FormLabel>{'Are you currently using any mobility aids?'}</FormLabel>
+            <RadioGroup
+              value={mobilityAidsRequired ? '1' : '0'}
+              onChange={value => {
+                setMobilityAidsRequired(value === '1' ? true : false);
+                if (value === '0') {
+                  onChange({
+                    ...physicianAssessment,
+                    mobilityAids: [],
+                  });
+                }
+              }}
+            >
+              <Stack>
+                <Radio value={'1'}>{'Yes, I am'}</Radio>
+                <Radio value={'0'}>{'No, I am not'}</Radio>
+              </Stack>
+            </RadioGroup>
+          </FormControl>
+
+          {mobilityAidsRequired === true && (
```

Comment:
Lmk if there's a cleaner way to handle this. The current approach is to use a state `mobilityAidsRequired` to determine whether or not to render the Other mobility aids text area. If the user selects that they do not have mobility aids the if statement is hit and the value of mobility aid is set to `[]`.

Reply:
Addressed!